fe/fe-core/src/main/java/com/starrocks/lake/compaction/CompactionMgr.java

```diff
@@ -119,7 +119,7 @@ public void handleCompactionFinished(PartitionIdentifier partition, long version
             }
             v.setCurrentVersion(compactionVersion);
             v.setCompactionVersion(compactionVersion);
-            v.setCompactionScore(compactionScore);
+            v.setCompactionScore(compactionScore, false /* isLoad */);
```

Comment:
if these two are the only places to call the `setCompactionScore`, I would suggest add new interface instead of adding new parameter.

`setCompactionScore` will just do the compaction score setter.
`setCompactionScoreAndAdjustPunishFactor` will do both.

The caller can choose whatever is proper.

Reply:
done